javascript.js

```diff
@@ -0,0 +1,7 @@
+window.location.hash="#start";
+let a = document.querySelectorAll("div[id]");
+a.forEach(function(e) {
+  if ("#" + e.getAttribute("id") != location.hash) {
+    e.style.display = "none";
+  }
+})
```

Comment:
i dont think this file (and logic) is being used at all
please delete unused code/files hehe